inst/schema/spec.md

```diff
@@ -0,0 +1,229 @@
+This is all highly uncertain and liable to change!
+
+# Data
+## POST /data/tab-name/
```

Comment:
As discussed on a call, let's not prematurely generalise on the similarities between the tabs or bake the front-end design into the backend api. We can duplicate the endpoint and share a response schema perhaps, which will give us some flexibility going forward to modify things